apps/admin-gui/src/app/app.component.ts

```diff
@@ -33,6 +40,8 @@ export class AppComponent implements OnInit {
   displayWarning: boolean = this.store.get('display_warning');
   warningMessage: string = this.store.get('warning_message');
 
+  version: string;
```

Comment:
The first version you use for a check should be the one, which was used during the build of the app. 
```suggestion
  version = version;
```
You can import this version like this:
```
import { version } from '../../../../package.json';
```